net.hypatiaproject.Hypatia.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "net.hypatiaproject.Hypatia",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "48",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "hypatia",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	"shared-modules/libsoup/libsoup-2.4.json",
```

Comment:
you didn't add the gitmodule here.

Reply:
I added it to the main repository, but forgot to add it here as well. Sorry about that - should be fixed.